security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -0,0 +1,75 @@
+
+[id="compliance-operator-release-notes-0-1-44"]
+== OpenShift Compliance Operator 0.1.44
+
+The following advisory is available for the OpenShift Compliance Operator 0.1.44:
+
+* link:https://access.redhat.com/errata/RHBA-2021:4530[RHBA-2021:4530 OpenShift Compliance Operator Bug Fix and Enhancement Update]
+
+[id="compliance-operator-0-1-44-new-features-and-enhancements"]
+=== New features and enhancements
+
+* In this release, the `strictNodeScan` option is now added to the `ComplianceScan`, `ComplianceSuite` and `ScanSetting` CRs. This option defaults to "true" which matches the previous behavior where an error occurred if a scan was not able to be scheduled on a node. Setting the option to "false" (meaning not strict, permissive) allows the Compliance Operator to be more lenient about scheduling scans. Environments with ephemeral nodes can set `strictNodeScan` to false which will allow a compliance scan to proceed, even if some of the nodes in the cluster are not available for scheduling.
++ 
+* You can now customize the nodes that are used to schedule the `ResultServer` workload by configuring the `nodeSelector` and `tolerations` attributes of the `ScanSetting` object. These attributes are used to place the `ResultServer` pod (a pod where the raw ARF results are uploaded to, the pod mounts a PV storage volume). Previously, the `nodeSelector` and the `tolerations` defaulted to selecting master nodes and tolerating the node-role.kubernetes.io/master taint. This did not work in environments where master nodes are not permitted to mount PVs. This feature provides another way for you to select nodes and tolerate different taints in those environments.  
++
+* The Compliance Operator can now remediate `KubeletConfig` objects.
++
+* A comment containing an error message is now added to help content developers differentiate between objects that do not exist in the cluster versus objects that cannot be fetched.
++
+* Rule objects now contain two new attributes, `checkType` and `description`. These attributes allow you to determine if the rule pertains to a node check or platform check, and also allows you to review what the rule does.
++
+* This enhancement removes the requirement that you have to extend an existing profile in order to create a tailored profile. This means the `extends` field in the `TailoredProfile` CRD is no longer mandatory. You can now select a list of rule objects to create a tailored profile from scratch.  Note: You will need to select whether your profile applies to nodes or the platform by setting the `compliance.openshift.io/product-type:` annotation or by setting the `-node` suffix for the `TailoredProfile` CR.
```

Comment:
"from scratch" is idiomatic, and, for better internationalization, the ISG says to avoid using idioms.

```suggestion
* This enhancement removes the requirement that you have to extend an existing profile in order to create a tailored profile. This means the `extends` field in the `TailoredProfile` CRD is no longer mandatory. You can now select a list of rule objects to create a tailored profile.  Note that you must select whether your profile applies to nodes or the platform by setting the `compliance.openshift.io/product-type:` annotation or by setting the `-node` suffix for the `TailoredProfile` CR.
```